From 6bb0e8aea27f6060a8d68dd264c5a2f00483d108 Mon Sep 17 00:00:00 2001
From: Sakura286 <sakura286@outlook.com>
Date: Tue, 4 Jun 2024 16:53:26 +0800
Subject: [PATCH 2/2] reverse std::vector patch

---
 src/registration_list.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/registration_list.cpp b/src/registration_list.cpp
index 74f82ee..6dec244 100644
--- a/src/registration_list.cpp
+++ b/src/registration_list.cpp
@@ -207,7 +207,7 @@ void RegistrationList::deleteElement(size_t index)
   ids_.resize(ids_.size() - 1);
   std::swap(references_[index], references_.back());
   references_.resize(references_.size() - 1);
-  std::vector<bool>::swap(enabled_[index], enabled_.back());
+  std::swap(enabled_[index], enabled_.back());
   enabled_.resize(enabled_.size() - 1);
 
   registrationsChanged();
-- 
2.34.1

